src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -16,21 +16,30 @@
  */
 
 #include "AreaTrigger.h"
+#include "CellImpl.h"
+#include "Chat.h"
 #include "DB2Stores.h"
+#include "GridNotifiersImpl.h"
 #include "Log.h"
+#include "Object.h"
+#include "ObjectMgr.h"
+#include "Player.h"
 #include "SpellInfo.h"
 #include "Unit.h"
 #include "UpdateData.h"
+#include "ScriptMgr.h"
 
-AreaTrigger::AreaTrigger() : WorldObject(false), _duration(0), _spellXSpellVisualId(0)
+AreaTrigger::AreaTrigger() : WorldObject(false), _duration(0), _spellXSpellVisualId(0), _timeSinceCreated(0)
 {
     m_objectType |= TYPEMASK_AREATRIGGER;
     m_objectTypeId = TYPEID_AREATRIGGER;
 
-    m_updateFlag = UPDATEFLAG_STATIONARY_POSITION;
+    m_updateFlag = UPDATEFLAG_STATIONARY_POSITION | UPDATEFLAG_AREATRIGGER;
 
     m_valuesCount = AREATRIGGER_END;
     _dynamicValuesCount = AREATRIGGER_DYNAMIC_END;
+
+    _areaTriggerTemplate = nullptr;
```

Comment:
Initialize this in L32 after `_timeSinceCreated` please (`_areaTriggerTemplate(0)`).